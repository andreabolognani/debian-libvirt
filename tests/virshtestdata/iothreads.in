iothreadinfo --domain fc4
iothreadadd --domain fc4 --id 6
iothreadinfo --domain fc4
iothreaddel --domain fc4 --id 2
iothreadinfo --domain fc4

domstats --domain fc4
iothreadset --domain fc4 --id 6 --poll-max-ns 100 --poll-shrink 10 --poll-grow 10
domstats --domain fc4

iothreadadd --domain fc5 --id 2
iothreadinfo --domain fc5
iothreadpin --domain fc5 --iothread 2 --cpulist 0
iothreadinfo --domain fc5
