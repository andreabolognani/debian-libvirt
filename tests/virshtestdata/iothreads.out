 IOThread ID   CPU Affinity
-----------------------------
 2             0
 4             0


 IOThread ID   CPU Affinity
-----------------------------
 2             0
 4             0
 6             0


 IOThread ID   CPU Affinity
-----------------------------
 4             0
 6             0

Domain: 'fc4'
  state.state=1
  state.reason=0
  iothread.count=2
  iothread.4.poll-max-ns=32768
  iothread.4.poll-grow=0
  iothread.4.poll-shrink=0
  iothread.6.poll-max-ns=32768
  iothread.6.poll-grow=0
  iothread.6.poll-shrink=0


Domain: 'fc4'
  state.state=1
  state.reason=0
  iothread.count=2
  iothread.4.poll-max-ns=32768
  iothread.4.poll-grow=0
  iothread.4.poll-shrink=0
  iothread.6.poll-max-ns=100
  iothread.6.poll-grow=10
  iothread.6.poll-shrink=10


 IOThread ID   CPU Affinity
-----------------------------
 2             0-3


 IOThread ID   CPU Affinity
-----------------------------
 2             0

